docs/management/managing-saved-objects.asciidoc

```diff
@@ -1,9 +1,9 @@
 [[managing-saved-objects]]
-== Saved objects
+== Saved Objects
 
 *Saved Objects* helps you keep track of and manage your saved objects. These objects
```

Comment:
Instead of starting all of these sections with "[Feature] helps you...", how about just starting with what the UI helps the user do? So for this one, it would be:

Keep track of and manage your saved objects.